doc/source/ray-core/doc_code/direct_transport_nixl.py

```diff
@@ -14,6 +14,22 @@ def random_tensor(self):
     def sum(self, tensor: torch.Tensor):
         return torch.sum(tensor)
 
+    def produce(self, tensors):
+        refs = []
+        for t in tensors:
+            refs.append(ray.put(t, _tensor_transport="nixl"))
+        return refs
+
+    def consume_with_nixl(self, refs):
+        # The :func:`ray.get <ray.get>` function will also use NIXL to retrieve the
+        # result.
```

Comment:
```suggestion
        # ray.get will use NIXL to retrieve the result
```
the `:func:`ray.get <ray.get>` doesn't show up in code comments

<img width="715" height="169" alt="Image" src="https://github.com/user-attachments/assets/0d12d8d6-3d6c-4ddd-bb0f-d6f0609f724f" />